canonical-web-and-design/ubuntu.com
# Defer scripts and remove redundant polyfill

## Done

Defer scripts and remove redundant polyfill

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check site contact form loads without any console race condition errors
